nevergrad/functions/images/core.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from math import sqrt, tan, pi
+import numpy as np
+import nevergrad as ng
+import PIL
+import PIL.Image
+import os
+from nevergrad.common.typetools import ArrayLike
+from .. import base
+
+
+class Image(base.ExperimentFunction):
+    def __init__(self, problem_type: str = "recovering", index_pb: int = 0) -> None:
```

Comment:
I'd rather use `problem_name` instead of `problem_type`: `type` seems too code-related to me
and either `problem_index` or `index`, it's weird to use a shortcut for one variable and not the other
Also, how many  indices do you plan to add? Adding a large number of files to the repo is not recommended

Reply:
Done, thx.

Reply:
you probably did not push then :p

Reply:
yeah sorry for that...